stdlib/experimental/influxdb/influxdb.flux

```diff
@@ -1,8 +1,63 @@
+// Package influxdb provides tools for working with the InfluxDB API. 
+// 
+// introduced: 0.114.0
+// 
 package influxdb
 
 
-// api submits an HTTP request to the specified API path.
-// Returns HTTP status code, response headers, and body as a byte array.
+// api submits an HTTP request to the specified InfluxDB API path and returns a
+// record containing the HTTP status code, response headers, and the response body.
+// 
+// ## Response format
+// `influxdb.api()` returns a record with the following properties:
+// 
+// - **statusCode**: HTTP status code returned by the GET request (int).
+// - **headers**: HTTP response headers (dict).
+// - **body**: HTTP response body (bytes). 
+// 
+// ## Parameters
+// - method: HTTP request method.
+// - path: InfluxDB API path.
+// - host: InfluxDB host URL _(Required when executed outside of InfluxDB)_.
+//   Default is `""`.
+// - token: [InfluxDB API token](https://docs.influxdata.com/influxdb/cloud/security/tokens/)
+//   _(Required when executed outside of InfluxDB)_.
+//   Default is `""`.
+// - headers: HTTP request headers.
```

Comment:
I just noticed that `headers` is a `dict` here and a `record` above... I wish we were consistent.

Reply:
Yeah, the `experimental/http` package was added before support for dictionaries. This package was added after. I'm all for consistency as well.